src/features/Dev/DevSlice.ts

```diff
@@ -0,0 +1,103 @@
+import { SagaIterator } from '@redux-saga/types';
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { takeEvery, getContext, all, call, put } from 'redux-saga/effects';
+
+import { Sensor, TemperatureLog } from '../../common/services/Database/entities';
+import { DEPENDENCY, REDUCER } from '../../common/constants';
+
+import { SensorAction } from '../Entities';
+import { ConsecutiveBreachAction } from '../Breach'
+import { TemperatureLogAction } from '../Entities';
+
+interface CreatePayload {
+    sensor: Sensor;
+  }  
+
+const initialState = {};
```

Comment:
could see some state could help with showing some feedback as to whats going on (future problem)